cranelift/Cargo.toml

```diff
@@ -49,6 +49,7 @@ similar = { workspace = true }
 toml = { workspace = true }
 serde = { workspace = true }
 fxhash = "0.2.1"
+regalloc2 = { workspace = true, features = ["trace-log"] }
```

Comment:
Note: this is just `clif-util` enabling the `trace-log` feature by default, not changing `cranelift-codegen` or any other crate that is actually used as a dep anywhere.